analogwp-templates.php

```diff
@@ -123,22 +123,52 @@ function analog_fail_load() {
 		return;
 	}
 
-	if ( ! function_exists( 'get_plugins' ) ) {
-		require_once ABSPATH . 'wp-admin/includes/plugin.php';
+	$file_path = 'elementor/elementor.php';
+
+	$is_not_activated = false;
+	$is_not_installed = false;
+
+	if ( version_compare( get_bloginfo( 'version' ), '5.5a', 'gt' ) ) {
```

Comment:
@mauryaratan the comparison check is installed WordPress version is greater than 5.5 alpha which results true for 5.5 beta, release candidate, general availability, and any other greater version. The time when we implemented this WordPress 5.5 was in beta stage so added the comparison with `5.5a` instead of `5.5b`.